ARExperiment/ARExperiment/RecognizeObjectsViewController.swift

```diff
@@ -3,33 +3,34 @@ import ARKit
 import SceneKit
 import Vision
 
-class RecognizeObjectsViewController: UIViewController, ARSCNViewDelegate, ARExperimentSessionHandler {
+class RecognizeObjectsViewController: UIViewController, ARExperimentSessionHandler {
     
     @IBOutlet weak var sceneView: ARSCNView!
     fileprivate let yolo = YOLO()
     private let semaphore = DispatchSemaphore(value: 2)
     private var request: VNCoreMLRequest!
     private let startButton = UIButton()
-    private let compoundingBox = UIView()
-    private let predictionLabel = UILabel()
     private let arViewModel = ARViewModel()
     private let nodeName = "cubewireframe"
     private let fileName = "cubewireframe"
     private let fileExtension = "dae"
+    private var viewSizeForScale: CGRect = .zero
```

Comment:
I don't think it's interesting to include the reason why we need a variable in its name. Also the name indicate that it's a size while it's a rect. 
From the code it seems that a better name for this variable should be `currentSceneFrame`

Reply:
done 😄